tests/test_lib_caconnector.py

```diff
@@ -280,6 +281,25 @@ def test_05_templates(self):
         self.assertTrue(ddiff.days > 740, ddiff.days)
         self.assertTrue(ddiff.days < 760, ddiff.days)
 
+    def test_99_cleanup(self):
+        filelist = glob.glob("{0!s}/100*.pem".format(WORKINGDIR))
+        for f in filelist:
+            os.remove(f)
+
+        FILES = ["cornelius_user@localhost.localdomain_realm1.pem",
+                 "cornelius_user@localhost.localdomain_realm1.req",
+                 "DE_Hessen_privacyidea_requester.localdomain.req",
+                 "DE_Hessen_privacyidea_usercert.pem",
+                 "DE_Hessen_privacyidea_usercert.req",
+                 "index.txt.attr.old",
+                 "index.txt.old",
+                 "serial.old",
+                 "crl.pem",
+                 "Steve_Test.der",
+                 "Steve_Test.txt"]
+        for f in FILES:
+            os.remove("{0!s}/{1!s}".format(WORKINGDIR, f))
```

Comment:
maybe we should use `try-except` here as well (catching `IOError` should be enough).

Reply:
Hm, if the test run successfully, the files should exist. Actually I *had* this in the first place but then realized that this would not be necessary and removed it before the commit, since it *looked nicer* ;-)

Reply:
will check